mapit/management/find_parents.py

```diff
@@ -46,7 +46,12 @@ def handle(self, **options):
                 except Area.DoesNotExist:
                     continue
             if not parent:
-                raise Exception("Area %s does not have a parent?" % (self.pp_area(area)))
+                # Fix for May 2019 August update
+                if self.pp_area(area) == 'WARREN [11574] (LGW)' and options['commit']:
+                    parent = Area.objects.get(name='Bangor East and Donaghadee') # id=11758 (LGE)
+                    area.save()
```

Comment:
I don't think we need this line as it should happen below on line 60:

```
area.parent_area = parent
area.save()
```

Reply:
Thanks @thomasleese ! I have addressed this in https://github.com/alphagov/mapit/pull/70/commits/5b2ede155a157d7d69883a6a0197513bcbcca4bb